### ISSUE TITLE
Dependency 'io.github.tweener:alarmee-android:1.3.1' requires core library desugaring to be enabled

### ISSUE THREAD

**Issue by 658jjh** [Tweener/alarmee#4]
I got 2 issues after implement the libs.
```
2 issues were found when checking AAR metadata:

  1.  Dependency 'io.github.tweener:alarmee-android:1.3.1' requires core library desugaring to be enabled
      for :composeApp.

      See https://developer.android.com/studio/write/java8-support.html for more
      details.

  2.  Dependency 'io.github.tweener:kmp-common-android:1.1.2' requires core library desugaring to be enabled
      for :composeApp.

      See https://developer.android.com/studio/write/java8-support.html for more
      details.
```

I just did the implementation like what you mentioned in the README, does it miss anything?

**Comment by Tweener** [comment]
As of version 1.3.1, Alarmee needs [desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring). Therefore, you need to add the core library desugaring dependency in your app. In your build.gradle.ktx:

```kotiln
android {
  ...

  compileOptions {
        ...

        isCoreLibraryDesugaringEnabled = true
    }

    dependencies {
        coreLibraryDesugaring("com.android.tools:desugar_jdk_libs:2.0.3")
    }
}
```

That being said, I'll make a new release which removes this library, as I don't think it is needed anymore. I'll do my best to release it ASAP.

**Comment by Tweener** [comment]
@658jjh I just released a new version, 1.3.2, of the library which removes the desugaring library.
This should fix your issues.
